Dockerfile

```
@@ -163,12 +151,17 @@ RUN \
         pulseaudio-alsa \
         socat
 
+WORKDIR /tmp/
+COPY requirements.txt /tmp/requirements.txt
+RUN \
+    --mount=type=cache,target=/root/.cache/pip,sharing=locked,id=pip-cache-${BUILD_FROM} \
+    pip3 install --only-binary=:all: \
+        -r /tmp/requirements.txt \
+    && rm -f /tmp/requirements.txt
```

It's slightly out of scope of the PR but we can save one useless layer here:
```suggestion
RUN \
    --mount=type=bind,src=./requirements.txt,dst=/tmp/requirements.txt \
    --mount=type=cache,target=/root/.cache/pip,sharing=locked,id=pip-cache-${BUILD_FROM} \
    pip3 install --only-binary=:all: \
        -r /tmp/requirements.txt
```